src/muse/timeslices.py

```diff
@@ -549,7 +549,7 @@ def convert_timeslice(
         index = index.set_names(f"finest_{u}" for u in index.names)
         mindex_coords = xr.Coordinates.from_pandas_multiindex(index, "finest_timeslice")
         finest = finest.drop_vars(list(finest.coords)).assign_coords(mindex_coords)
-        proj0 *= finest
+        proj0 = proj0 * finest
```

Comment:
Ok... but why this works and `*=` doesn't when the types are different?

Reply:
My limited understanding was that a *= b is more memory efficient as it doesn't create a copy of a in the process. However, if a and b are different types (e.g. int vs float) it can't process. Meanwhile, a = a*b is less efficient but can work with different types because it can convert when it copies a to carry out the multiplication.

However, this may be wrong as it is based on reading between the lines of moderately-helpful documentation.